examples/function_memory_game/src/components/chessboard_card.rs

```diff
@@ -0,0 +1,54 @@
+use web_sys::MouseEvent;
+use yew::prelude::*;
+
+use crate::constants::CardName;
+use crate::state::{Card, RawCard};
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    pub card: Card,
+    pub on_flip: Callback<RawCard>,
+}
+
+#[function_component]
+pub fn ChessboardCard(props: &Props) -> Html {
+    let Card { flipped, name, id } = props.card.clone();
+
+    let get_link_by_cardname = {
+        match name {
+            CardName::EightBall => "public/8-ball.png",
```

Comment:
Putting this here because I can't put it on the file directly.

Do we know the origin of these image assets? Or more importantly, the license?

Reply:
This is port version of [Memory-Game](https://github.com/bradlygreen/Memory-Game). I will try to check with the author if it is possible we can use them Since he forgot to specify the license in his project. 
Thanks for reminding me the issue

Reply:
@siku2  i got confirmed by the author, https://github.com/bradlygreen/Memory-Game/issues/6. We are possible to use those images

Reply:
Thanks for confirming